contracts/YearnHarvest.sol

```diff
@@ -175,6 +177,22 @@ contract YearnHarvest {
         uint256 callCostInWei = 1e8; // low value so that the trigger focuses on all other conditions
 
         // Check active strategies and return the first one that is ready to harvest.
+
+        // REVIEW: Two issues of iterating over ALL strategies I can think of:
+        // 1) You might run out of gas. If there are enough reverting harvest
```

Comment:
This function is not meant to be ran as a tx, but as a static call. In the article I explain that Gelato calls it with a static-call, which I think it's smart as checking the conditions on chain could be an expensive tx. This is why I don't write this function with gas as a main concern.